### ISSUE TITLE
Error Handling - Itemize all Error Pages Needed

### ISSUE THREAD

**Issue by megmay** [PranovD/LLOV#53]
[] Figure out all of the different types of error pages we should have (all of the ways the user could mess something up) and aggregate them on a list. (Errors such as unauthorized, bad request, payment required, etc.)

[] Add to the list all errors that could be thrown by Plaid, MailChimp, and any other external APIs we're using. Some of these errors could end up on the same error page, just because Plaid & Mailchimp throw a page not found error doesn't mean that they each need their own error page. However, sometimes they will need separate error pages, it depends on what the user did that needs correcting.

errors.py and error_handlers.py has a couple of examples of error implementation
Basically, go through these and pick out the errors that seem suitable (in addition to writing down the error name, should also write down the status code associated with it): https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#4xx_Client_errors
